### ISSUE TITLE
QuantumFlow: Similar project?

### ISSUE THREAD

**Issue by stavros11** [qiboteam/qibo#49]
I accidentally found [QuantumFlow](https://github.com/gecrooks/quantumflow-dev) yesterday, which seems to be similar to QIBO in the sense that they simulate circuits using various backends. They have a numpy and a Tensorflow backend. I have not tried using it yet but the idea seems very similar to QIBO and they also say that Tensorflow can be used for gradient descent, etc. It seems that it is an old Rigetti project and the code looks much more complicated than QIBO. We may want to check how similar we are and also run some benchmarks.
